packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.tsx

```diff
@@ -197,6 +197,7 @@ export class DetailsRow extends React.Component<IDetailsRowProps, IDetailsRowSta
         data-selection-index={ itemIndex }
         data-item-index={ itemIndex }
         data-is-draggable={ isDraggable }
+        draggable= { isDraggable }
```

Comment:
We're not using data-is-draggable anywhere else. Not sure if we can remove it. draggable is the html5 property to enable an element to be dragged.